src/main/java/com/dropbox/core/oauth/DbxCredential.java

```diff
@@ -163,13 +163,14 @@ public boolean aboutToExpire() {
      * but just add host.
      * @param requestConfig request config used to make http request.
      * @param host which host to call, only used in internal testing.
+     * @param scope space-delimited scope list. Must be subset of scope in current oauth2 grant.
```

Comment:
I think it would be easier for the end client to pass in a list of strings and then join them rather than relying on them space delimiting themselved

Reply:
Changed. Also change all there function interface to take Collection<String>